src/utilities/effects.ts

```diff
@@ -4723,7 +4722,7 @@ const effects = {
           }
         }
       }
-      return generateDefaultView(activityTypes, resourceTypes, externalEventTypes);
+      return generateDefaultView(resourceTypes, externalEventTypes);
```

Comment:
Is it okay that `resourceTypes` is being passed into `generateDefaultView` as the `activityTypes` parameter?

Reply:
generateDefaultView was also changed to not accept `activityTypes` as an argument

Reply:
I'm dumb...my eyes couldn't see the red indicating it was removed for some reason...